.github/workflows/ci.yml

```
@@ -88,6 +93,10 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version:  ${{ matrix.node-version }}
+      - name: Run foreman nmp install
```

nitpick:
```suggestion
      - name: Run foreman npm install
```
also on line 100

---

thanks :+1: I've extracted the GH action changes to #18 and merged.